tests/e2e/utils/data_fetcher/logs.py

```diff
@@ -0,0 +1,39 @@
+import json
+from functools import lru_cache
+from typing import List, Optional, Union
+
+from mypy_boto3_cloudwatch import CloudWatchClient
+from pydantic import BaseModel
+from retry import retry
+
+
+class Log(BaseModel):
+    level: str
+    location: str
+    message: Union[dict, str]
+    timestamp: str
+    service: str
+    cold_start: Optional[bool]
+    function_name: Optional[str]
+    function_memory_size: Optional[str]
+    function_arn: Optional[str]
+    function_request_id: Optional[str]
+    xray_trace_id: Optional[str]
+    extra_info: Optional[str]
+
+
+@lru_cache(maxsize=10, typed=False)
```

Comment:
Should we remove this now?

Reply:
It will be in the next PR specifically for Logger